### PR TITLE
Add Impact Index description to seed data

### PR DESCRIPTION
**Pull request by escottalexander**
Just adding the Impact Index description to seed data.

When this gets merged I will update the database.

### CODE REVIEW

File: packages/nextjs/services/mongodb/seed.sample.json [hunk 1]
```diff
@@ -21842,7 +21842,7 @@
       "label": "Impact Index",
       "longLabel": "Impact Index",
       "name": "impact_index",
-      "description": "string",
+      "description": "The Impact Index is based on the results of RF4 in which the badgeholder's votes created an average weight for each metric used in that round. The top five metrics were Gas Fees (31.5% when including the logarithmic Gas Fee metric), Trusted Users Onboarded (12.5%), Transaction Count (9.5% when including the logarithmic Transaction Count metric), Trusted Recurring Users (8.7%), Trusted Monthly Active Users (6.5%). There was also additional rewards based on projects that qualified for a set of Open Source criteria that is not expressed in the Impact Index.",
```
Comment (swellander) [inline packages/nextjs/services/mongodb/seed.sample.json:21845]: Could maybe add an `and` between "Trusted Recurring Users (8.7%)," and "Trusted Monthly Active Users (6.5%)"

Comment (swellander) [reply packages/nextjs/services/mongodb/seed.sample.json:21845]: "There ~was~ were also additional rewards"

**Review by swellander — DISMISSED** [review]
Just left a few comments on the text, but otherwise LGTM!

File: packages/nextjs/services/mongodb/seed.sample.json (hunk 1 above)
Comment (escottalexander) [reply packages/nextjs/services/mongodb/seed.sample.json:21845]: Thanks for this feedback. Addressed in latest commit.

**Pull request merged**